src/style.css

```
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background: no-repeat url(images/star-active.svg);
+  background-position: 50% 50%;
```

Better to use just center, it's the same
```suggestion
  background-position: center;
```

---

Done